### ISSUE TITLE
Unlock pattern dots too agressive

### ISSUE THREAD

**Issue by AlkalineJaunt** [TeamWin/Team-Win-Recovery-Project#1398]
- [x] I am running an official build of TWRP, downloaded from https://twrp.me/Devices/
- [x] I am running the latest version of TWRP
- [x] I have read the FAQ (https://twrp.me/FAQ/)
- [x] I have searched for my issue and it does not already exist

**Device codename**: bullhead
**TWRP version**: 3.2.3-0

#### WHAT STEPS WILL REPRODUCE THE PROBLEM?

It is impossible to squeeze between horizontal/vertical dots in a 5x5 pattern without one of them angrily grabbing the unlock pattern.

#### WHAT IS THE EXPECTED RESULT?

Users should be able to connect arbitrary dots by sliding past unwanted dots.

#### WHAT HAPPENS INSTEAD?

Nearby dots grab the pattern. It looks like their area of sensitivity is too large.

**Comment by Heidistein** [comment]
Especially with my thick fingers on a 6x6 pattern using diagonal lines this is a hassle...

**Comment by CaptainThrowback** [comment]
I assume the code that would need to be updated is in [here](https://github.com/omnirom/android_bootable_recovery/blob/android-9.0/gui/patternpassword.cpp) somewhere, but I have no idea what exactly would need to be adjusted. If you can figure it out and submit a patch to Gerrit, I'm sure it would be reviewed.

**Comment by AlkalineJaunt** [comment]
I'd wager it's mDotRadius*3 from here:
https://github.com/TeamWin/Team-Win-Recovery-Project/blob/58f2132bc3954fc704787d477500a209eedb8e29/gui/patternpassword.cpp#L268

**Comment by Heidistein** [comment]
Yes, i was looking at that, thinking about to try something like, first making the sots smaller, perhaps dynamic depending on the gridsize. Next I was stuck at compiling it, and ran out of time.
